scijava-function/templates/main/java/org/scijava/function/Inplaces.vm

```diff
@@ -167,5 +168,17 @@ public final class Inplaces {
 		public int mutablePosition() {
 			return mutablePosition;
 		}
+
+		@Override
+		public boolean equals(Object that) {
```

Comment:
Should this addition of `equals` to `Inplaces` be done as a separate commit? It doesn't seem relevant to the commit's stated goal.

Reply:
Probably - made it a separate commit (6d5ec93)